docs/source/_ext/edit_on_github.py

```diff
@@ -8,19 +8,22 @@
 import os
 import warnings
 
-__licence__ = 'BSD (3 clause)'
+__licence__ = "BSD (3 clause)"
```

Comment:
Please limit your changes only to the relevant integration.

Reply:
It was changed by black. I have changed it to the original state.